sdk/servicebus/azure-messaging-servicebus/src/main/java/com/azure/messaging/servicebus/ServiceBusErrorSource.java

```diff
@@ -0,0 +1,27 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+package com.azure.messaging.servicebus;
+
+/**
+ * Represent the operation user was performing when the error happened.
+ */
+public enum ServiceBusErrorSource {
+    /** Error while sending the message(s).*/
+    SEND,
```

Comment:
Are we using this exception for the sender client too?

Reply:
yes we are doing this for Send also.